x-pack/solutions/search/plugins/enterprise_search/server/utils/search_result_provider.test.ts

```diff
@@ -172,30 +141,7 @@ describe('Search search provider', () => {
             mockSearchProviderContext
           )
         ).toBe('(a|)', {
-          a: [{ ...crawlerResult, score: 80 }],
-        });
-      });
-    });
-
-    it('omits crawler if config has crawler disabled', () => {
-      const searchProvider = getSearchResultProvider(
-        {
-          hasConnectors: true,
-          hasWebCrawler: false,
-        } as any,
-        connectors,
-        false,
-        'crawlerIcon.svg'
-      );
-      getTestScheduler().run(({ expectObservable }) => {
-        expectObservable(
-          searchProvider.find(
-            { term: '' },
-            { aborted$: NEVER, maxResults: 100, preference: '' },
-            mockSearchProviderContext
-          )
-        ).toBe('(a|)', {
-          a: expect.not.arrayContaining([{ ...crawlerResult, score: 80 }]),
+          a: [{ score: 80 }],
```

Comment:
same as below. If you want to check what is returned., update this object with whatever you are expecting.